Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasNodeable_Header.jinja

```diff
@@ -107,6 +107,9 @@ public: \
 {% else %}
               /* no slot configuration extension, Use Class attribute 'ExtendConfigureSlots' to extend them */
 
+{% endif %}
+{% if Class.attrib['Style'] is defined %}
+              void OnInit() override;
```

Comment:
I think we can leave the OnInit() there by default, and then move the condition to the implementation